packages/block-editor/src/components/block-switcher/index.js

```diff
@@ -218,54 +214,48 @@ export const BlockSwitcherDropdownMenu = ( { clientIds, blocks } ) => {
 						} }
 						menuProps={ { orientation: 'both' } }
 					>
-						{ ( { onClose } ) =>
-							showDropDown && (
```

Comment:
All these are whitespace changes because I removed the `showDropDown` check, since we know we should render a Dropdown.